proxied/proxied.go

```diff
@@ -600,6 +600,19 @@ func DirectThenFrontedClient(timeout time.Duration) *http.Client {
 	}
 }
 
+func ChainedThenDirectThenFrontedClient(timeout time.Duration, rootCA string) *http.Client {
+	drt := &http.Transport{
+		TLSHandshakeTimeout:   timeout,
+		ResponseHeaderTimeout: timeout,
```

Comment:
We could actually hard-code this to something low, like 10 or 30 seconds.  That would allow us to move on to domain fronting faster if we're not able to hit the URL directly.

Reply:
I've hardcoded some timeouts but the client timeout I'll follow the recommendation for using some of the configs.